presto-docs/src/main/sphinx/release/release-0.253.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.253
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
+* Add tracking for system memory used by column statistics in TableFinishOperator.
+* Add support for shutting down coordinator via `/v1/info/state` endpoint.
+* Add :func:`binomial_cdf` and :func:`inverse_binomial_cdf` functions.
+
+Security Changes
+----------------
+* Add functionality to set schema access rules in file-based system access control
+
+Hive Changes
+____________
+* Fix S3 table creation error when the specified directory location is created from AWS console.
```

Comment:
Is this is a Hive change?

Reply:
It's a bug fix, and typically S3 metadata is stored in Hive, so seems to belong here.

Reply:
keeping it as per @tdcmeehan suggestion